src/charts/TileMap.js

```diff
@@ -32,7 +32,7 @@ function TileMap( props ) {
       }
     },
     series: [ {
-      type: 'map',
+      type: 'tile_map',
```

Comment:
This is a Highcharts setting and should stay map http://api.highcharts.com/highmaps/series%3Cmap%3E